dsc/DscForEngineers.md

```diff
@@ -1,55 +1,70 @@
 ---
-ms.date:  2017-06-12
-author:  eslesar
+ms.date:  2017-10-13
+author:  eslesar;mgreenegit
 ms.topic:  conceptual
 keywords:  dsc,powershell,configuration,setup
 title:  Desired State Configuration Overview for Decision Makers
 ---
 
-# Desired State Configuration Overview for Engineers #
+# Desired State Configuration Overview for Engineers
 
-This document is intended for developer and operations teams to understand the benefits of PowerShell Desired State Configuration (DSC).
-For a higher level view of the value DSC provides, please see [Desired State Configuration Overview for Decision Makers](decisionMaker.md)
+This document is intended for developer and operations teams
+to understand the benefits of PowerShell Desired State Configuration (DSC).
+For a higher level view of the value DSC provides,
+please see [Desired State Configuration Overview for Decision Makers](decisionMaker.md)
 
 ## Benefits of Desired State Configuration
 
 DSC exists to:
+
 - Decrease the complexity of scripting in Windows
 - Increase the speed of iteration
 
-The concept of "continuous deployment" is becoming more important. 
+The concept of "continuous deployment" is becoming more important.
```

Comment:
I'm aware i'm the original author of this line, but in hindsight it is not very useful and I think we can just remove it altogether :)